package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "twd",
-  "version": "1.3.0",
+  "version": "1.3.1",
```

Comment:
Also revert this, please. I update the version numbers myself as part of the release / packaging process.